### PR TITLE
Add codeowners

### PR DESCRIPTION
**Pull request by xvello**
As an experiment, add [code owners](https://help.github.com/articles/about-codeowners/) for several packages. +# These owners will be automatically added as reviewers for PRs, to ensure code quality and consistency of the package, and identify possible side effects.

PRs should still be peer-reviewed by the team opening the PR.

See https://github.com/DataDog/datadog-agent/pull/794 for an example. **Note:** code owners are added after the PR is created, not suggested in the PR creation form.

Please feel free to push on the branch to add/remove packages for your team, or open a review.

### CODE REVIEW

**Review by tmichelet — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+/pkg/tagger/                            @DataDog/kenafeh
+/pkg/util/docker/                       @DataDog/kenafeh @DataDog/burrito
+/pkg/util/ecs/                          @DataDog/kenafeh @DataDog/burrito
+/pkg/util/kubernetes/                   @DataDog/kenafeh @DataDog/burrito
+/pkg/collector/listeners/docker*        @DataDog/kenafeh
+/pkg/collector/corechecks/containers/   @DataDog/kenafeh
+
+/cmd/                                   @DataDog/croissant
+/pkg/aggregator/                        @DataDog/croissant
+/pkg/collector/                         @DataDog/croissant
```
Comment (hkaj) [inline .github/CODEOWNERS]: Could we make it more fine grained here? autodiscovery, listeners, providers is kenafeh's I think.
If you write
```
/pkg/collector/                         @DataDog/croissant
/pkg/collector/autodiscovery @Datadog/kenafeh
...
```
It should still make croissant the owners of everything in `collector` that is not explicitly listed below it.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+/pkg/forwarder/                         @DataDog/croissant
+/pkg/metadata/                          @DataDog/croissant
```
Comment (hkaj) [inline .github/CODEOWNERS]: Same here we should own ecs and kubernetes.

File: .github/CODEOWNERS (hunk 1 above)
Comment (xvello) [reply .github/CODEOWNERS]: The matching order is top to bottom, so we need to list the subdirs before. but indeed, I'll add autodiscovery to kenafeh

File: .github/CODEOWNERS (hunk 2 above)
Comment (xvello) [reply .github/CODEOWNERS]: /pkg/metadata/ecs | kubernetes are owned by burito at the top of the files. These just serialize the information to a burito-specific playload. The connection/parsing logic is in /pkg/util/ecs | kubernetes which both teams own

File: .github/CODEOWNERS (hunk 1 above)
Comment (hkaj) [reply .github/CODEOWNERS]:
```*       @global-owner1 @global-owner2

# Order is important; the last matching pattern takes the most
# precedence. When someone opens a pull request that only
# modifies JS files, only @js-owner and not the global
# owner(s) will be requested for a review.
*.js    @js-owner```

As I understand it Kenafeh should be after the general `/pkg/collector`. Maybe I'm mis-interpreting?

Comment (xvello) [reply .github/CODEOWNERS]: 👍  my bad

**Review by olivielpeau — COMMENTED** [review]
Nice stuff, thanks!

I think we can add Croissant as owner of `/omnibus/` too

**Review by conorbranagan — APPROVED** [review]
Looks good from our side

**Pull request merged**